src/calculateRentalCost.js

```diff
@@ -1,10 +1,22 @@
+const carCosts = 40;
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return carCosts * days - 50;
+  }
+
+  if (days >= 3) {
+    return carCosts * days - 20;
+  }
+
+  if (days < 3 && days > 0) {
```

Comment:
Prefer using `else if` for the last condition or simply return the last expression since the previous `if` statements contain `return` statements. This will simplify the function flow and improve readability.